extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -92,11 +89,15 @@ public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Co
                 for (ScheduledMethodMetadata method : context.getScheduledMethods()) {
 
                     invokers.put(method.getInvokerClassName(), context.createInvoker(method.getInvokerClassName()));
+                    AtomicInteger triggerNameSequence = new AtomicInteger();
```

Comment:
Hm, we don't need `AtomicInteger` here, right? We could use `int` and `triggerNameSequence++`...